coding-guides/tracking-used-invites.md

```diff
@@ -26,9 +26,6 @@ const invites = {};
 const wait = require('util').promisify(setTimeout);
 
 client.on('ready', async () => {
-  // "ready" isn't really ready. We need to wait a spell.
-  await wait(1000);
```

Comment:
Not sure why this was removed, as you're giving the application more than enough time to finish getting ready before caching all the invites.

Reply:
I'll just fix it, my bad!